src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
It's good practice to follow encapsulation principles. The 'reportHead' and 'reportDataDelimiter' should be private to prevent unwanted modification from outside of the class.

Reply:
++